examples/spec/lcdproc/LCDd.ini

```diff
@@ -0,0 +1,2752 @@
+[]
+mountpoint = LCDd.conf
+;infos/plugins = dump validation enum range ipaddr path network type
+infos/plugins = dump validation enum
+
+;Until the max plugin error is not fixed, some specifications will be left commented out
+
+;=============================================================================
+;=============================================================================
+
+;Affecting all subsettings, eg curses/Contrast, CFontz/Contrast, etc.
+;------------------------------------
+
+[/_/contrast]
+type = unsigned_short
+check/type = unsigned_short
```

Comment:
needed?

Reply:
Contrast can only range from 1-1000. If the range plugin automatically checks for the type too it doesn't matter i guess.

Reply:
`type` matters, as it will be the type @kodebach gets. `check/type` should not be needed here.

Reply:
One note on types: the highlevel API only uses `type`. The `type` plugin meanwhile will prefer `check/type` but falls back to `type`. This means in most cases just `type` is enough, but `check/type` can be used for overrides (e.g. for enums without codegen)

Reply:
@Piankero so you do not need "check/type" if it is the same as "type". Can you also add this as documentation to the "type" plugin?

Reply:
https://github.com/ElektraInitiative/libelektra/pull/2475